### PR TITLE
flatpak: add pt_BR translation

### PR DESCRIPTION
**Pull request by gugacavalieri**
- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

**Version of the command being documented (if known):**

### CODE REVIEW

**Review by navarroaxel — APPROVED** [review]

File: pages.pt_BR/linux/flatpak.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# flatpak
+
+> Construa, instale e execute aplicações e plataformas flatpak.
+> Mais informações: <https://docs.flatpak.org/en/latest/flatpak-command-reference.html#flatpak>.
+
+- Executar uma aplicação instalada:
+
+`flatpak run {{name}}`
```
Comment (marchersimon) [inline pages.pt_BR/linux/flatpak.md:8]: Could you please also translate all arguments if possible? Thanks

**Comment by guga-kudos** [comment]
Sure @marchersimon! I will address the suggestion and have a look at #6509 later by EOD :smile:

**Comment by gugacavalieri** [comment]
@marchersimon, comments addressed on this PR.

Opened https://github.com/tldr-pages/tldr/pull/6798 to address #6509.

**Review by marchersimon — APPROVED** [review]
Thanks, looks good

**Pull request merged**